pkg/storaged/client.js

```diff
@@ -1138,10 +1138,10 @@
 function stratis2_fetch_manager_properties(proxy) {
     stratis2_fetch_properties(proxy, ["LockedPoolsWithDevs"]).then(values => {
         if (values.LockedPoolsWithDevs) {
-            proxy.LockedPools = { };
+            proxy.StoppedPools = { };
             for (const uuid in values.LockedPoolsWithDevs) {
                 const l = values.LockedPoolsWithDevs[uuid];
-                proxy.LockedPools[uuid] = {
+                proxy.StoppedPools[uuid] = {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18679-20230430-073933-28ce209d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)